data/Viral kinetics of SARS-CoV-2/data.yaml

```diff
@@ -0,0 +1,105 @@
+analytes:
+  throatswab_SARSCoV2:
+    biomarker: SARS-CoV-2
+    description: SARS-CoV-2 RNA genome copy concentration in throat swab samples.
+      Specimens were collected from all patients at least 2 days after hospitalization
+      and physicians checked their symptoms and signs daily.
+    gene_target: unknown
+    limit_of_detection: unknown
+    limit_of_quantification: 1000
+    reference_event: PatientID
+    specimen: throat_swab
+    unit: value
+description: The authors measured SARS-CoV-2 in longitudinal throat swab samples collected
+  from 71 COVID-19 patients between February 4 and April 7, 2020. Abundances were
+  quantified using diagnosed by real-time reverse transcription polymerase chain reaction
+  (RT-PCR). Specimens were collected from all patients at least 2 days after hospitalization
+  and physicians checked their symptoms and signs daily. Patients who had asymptomatic
+  carrier and incubation period were analyzed.
+doi: 10.1016/j.ijid.2020.04.083
+participants:
+- attributes:
+    age: 30.0
+    day: -2
+    estimated: 1
+    sevmax: 2.0
+    sex: male
+  died: 0
+  measurements:
+  - analyte: throatswab_SARSCoV2
+    time: -2
+    value: 1630516385.0
+  - analyte: throatswab_SARSCoV2
+    time: 1
+    value: 2773733.99
+  - analyte: throatswab_SARSCoV2
+    time: 3
+    value: 19462.36623
+  - analyte: throatswab_SARSCoV2
+    time: 5
+    value: 80276.20464
+  - analyte: throatswab_SARSCoV2
+    time: 8
+    value: 9582.968159
+  - analyte: throatswab_SARSCoV2
+    time: 10
+    value: 2323.319055
+  - analyte: throatswab_SARSCoV2
+    time: 11
+    value: 1.0
```

Comment:
Those values as 1 should be `negative` (which are those samples with CT=40 in Figure 1).